### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ThichNguyenSTS**
Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,60 @@
...
+![Alt text](./assets/MavenHome.png?raw=true "JAVA_HOME")
+
+4. Open this base project
 5. Project structure settings:
 - From the main menu of IntelliJ select: `File => Project Structure => Project Settings => Project`
 - Select the wanted SDK
+
+![Alt text](./assets/ProjectSetings.png?raw=true "JAVA_HOME")
+
 6. Select Maven plugin for IntelliJ:
```
Comment (SaigonTechnologyQC) [inline README.md:30]: Bổ sung thêm step check JAVA_HOME:
- Open terminal/cmd and type command "echo %JAVA_HOME%"

File: README.md [hunk 2]
```diff
@@ -1,34 +1,60 @@
...
 - Create `JAVA_HOME` environment variables
 - Add `JAVA_HOME` to Path: `%JAVA_HOME%\bin`
-3. Download and unzip latest Maven source file: https://dlcdn.apache.org/maven/maven-3
+
+![Alt text](./assets/JavaHome.png?raw=true "JAVA_HOME")
+
+3. Download and unzip latest Maven source file: [Link download](https://dlcdn.apache.org/maven/maven-3)
 - Create `MAVEN_HOME` environment variables
 - Add `MAVEN_HOME` to Path: `%MAVEN_HOME%\bin`
-4. Open existing project or create new one
```
Comment (SaigonTechnologyQC) [inline README.md:11]: Bổ sung thêm step check MAVEN_HOME:
- Open terminal/cmd and type command "echo %MAVEN_HOME%"

File: README.md [hunk 3]
```diff
@@ -1,34 +1,60 @@
...
 - Select the Maven plugin at `C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.1\plugins\maven\lib\maven3\bin`
 - Add the path of Maven plugin above to the Path environment variables
-7. Download and unzip latest Allure report source file: https://docs.qameta.io/allure/
+
+![Alt text](./assets/MavenPlugins.png?raw=true "MavenPlugins")
+
+7. Download and unzip latest Allure report source file: [Link download](https://repo.maven.apache.org/maven2/io/qameta/allure/allure-commandline/)
 - Create `ALLURE_HOME` environment variables
 - Add `ALLURE_HOME` to Path: `%ALLURE_HOME%\bin`
+
```
Comment (SaigonTechnologyQC) [inline README.md:40]: Bổ sung thêm step check ALLURE_HOME:
- Open terminal/cmd and type command "echo %ALLURE_HOME%"

File: README.md (hunk 1 above)
Comment (ThichNguyenSTS) [reply README.md:30]: Fixed

File: README.md (hunk 3 above)
Comment (ThichNguyenSTS) [reply README.md:40]: Fixed

File: README.md (hunk 2 above)
Comment (ThichNguyenSTS) [reply README.md:11]: Fixed

**Pull request merged**